pkg/broker/ingress/multi_topic_decouple_sink.go

```diff
@@ -71,6 +71,10 @@ func (m *multiTopicDecoupleSink) getTopicForBroker(ns, broker string) (string, e
 		m.logger.Warn("config is not found for", zap.Any("ns", ns), zap.Any("broker", broker))
 		return "", fmt.Errorf("%q/%q: %w", ns, broker, ErrNotFound)
 	}
+	if brokerConfig.State != config.State_READY {
+		m.logger.Debug("broker is not ready", zap.Any("ns", ns), zap.Any("broker", broker))
+		return "", fmt.Errorf("%q/%q is not ready: %w", ns, broker, ErrNotFound)
```

Comment:
nit: create a new error `ErrNotReady`

Reply:
Sure. Do you expect a different http status code in that case?

Reply:
Probably, how about `503 Service Unavailable`? This implies that the unavailability is temporary: https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

Reply:
By 404 I meant it to mean something like "ready broker xxx is not found". But now I think 503 is better.